Assets/RGLUnityPlugin/Scripts/LowLevelWrappers/RGLMeshObject.cs

```diff
@@ -103,6 +105,22 @@ protected void UploadToRGL()
                 }
             }
         }
+
+        public void SetIntensityTexture(RGLTexture texture)
+        {
+            unsafe
+            {
+                try
+                {
+                    RGLNativeAPI.CheckErr(
+                        RGLNativeAPI.rgl_entity_set_intensity_texture(rglEntityPtr, texture.rglTexturePtr));
+                }
+                catch (RGLException)
+                {
+                    Debug.LogWarning($"Cannot assign texture to entity.");
+                }
```

Comment:
I'd enrich the log message with the entity's identifier. Also, I'd change the logging level to error instead of a warning.
Please remember about rethrowing.